opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -123,6 +124,33 @@ void groupLoop(const std::vector<const Opm::Group*>& groups,
     }
 }
 
+template <typename GroupOp>
+void groupLoop(const std::vector<const Opm::Group*>& groups,
+               GroupOp&&                             groupOp,
+               const Opm::ScheduleState&             sched_state,
+               const std::string&                    lgr_tag)
```

Comment:
I prefer that the call-back operation (here `groupOp`) be the last argument to these kinds of generic algorithms.